web-portal/backend/src/utils/utils.service.ts

```diff
@@ -124,6 +137,41 @@ export class UtilsService {
     }
     const data = await res.json();
     return data;
-  };
+  }
+
+  @Cron(CronExpression.EVERY_MINUTE)
+  async watchEvent() {
+    console.log('Getting Event Logs');
+    const blockNumber = await viemClient.getBlockNumber();
+    const logs = await viemClient.getLogs({
+      event,
+      fromBlock: blockNumber - BigInt(1000),
+      toBlock: blockNumber,
+    });
+
+    const parsedLogs: IParsedLog[] = logs.map((log: any) => ({
+      tenantId: web3.utils
+        .toAscii(log?.args?._identifier!)
+        .replaceAll(`\x00`, ''),
```

Comment:
What is going on here?

Reply:
Just in case somebody does redeem from etherscan with padded zeros to right